oresat_configs/scripts/sdo_transfer.py

```diff
@@ -33,15 +33,15 @@ def build_parser(parser: ArgumentParser) -> ArgumentParser:
         metavar="VALUE",
         nargs="?",
         default="",
-        help="data to write or for only octet/domain data types a path to a file "
-        "(e.g. file:data.bin)",
+        help="Data to write or for only octet/domain data types a path to a file."
+        " (e.g. file:data.bin) (Default: %(default)s)",
```

Comment:
This one doesn't really make sense to print the default value since it's the empty string.

Reply:
remove